### ISSUE TITLE
[BUG] Error in console

### ISSUE THREAD

**Issue by LoneDev6** [Thatsmusic99/HeadsPlus#21]
## FIRST-HAND CHECK - ARE YOU USING THE LATEST VERSION/SNAPSHOT OF HEADSPLUS? Have you checked to see if others have submitted the same bug report? 
State here when you have checked the following points above.

### Description of the bug
Idk, I just get the error in console, maybe it helps you fixing something

### To Reproduce
unknown, random

### Have you been able to reproduce the bug more than once?
yes, happened multiple times

### Expected behavior
no error

### Has a debug report been generated? 
https://hasteb.in/cusukuqi.json

### Required information
```Json
    "Server version": "git-Paper-282 (MC: 1.15.2)",
    "HeadsPlus version": "6.10.4",
    "Plugin values": {
      "Levels enabled": true,
      "Cached players": 13,
      "Total levels": 36,
      "Droppable heads enabled": true,
      "Player death messages": true,
      "Total challenges": 1891,
      "Autograb enabled": false,
      "MySQL is enabled": false,
      "Masks enabled": true,
      "Uses leaderboards": true,
      "Allows looting enchantment": true,
      "Uses heads selector": false,
      "Sellable heads enabled": true,
      "Stops placement of sellable heads": false
    },
    "NMS version": "v1_15_R1",
```

**Comment by Thatsmusic99** [comment]
This is one of those occasions where I get a bug report right before I head to bed so I will be responding again in the morning.

I will double check the plugin for the source of this error as whilst there is currently a snapshot release of v6.10.5-SNAPSHOT-1 (I may be making a full release on Spigot tomorrow), I hadn’t fixed a problem of the such; can you send me your mobs.yml file to check?

Thanks in advance!

**Comment by LoneDev6** [comment]
Don't worry, you work hard and you can take how much time you need :D
here is:
https://hasteb.in/elezovox.yaml

**Comment by Thatsmusic99** [comment]
Okay, it does look like you’re using the default configuration, so that does make it easier to identify, however I will be sure to check for any differences tomorrow just to be safe.

As for the error itself, are there any more of those? I have checked the stacktrace but I have no idea if it’ll do the same thing over and over. In this case, it was thrown when a monster was killed by a tipped arrow, but I have no clue what that monster is.

**Comment by LoneDev6** [comment]
An easy way would be to print the entity type when the exception occurs

Il ven 15 mag 2020, 23:52 TM (Holly) <notifications@github.com> ha scritto:

**Comment by Thatsmusic99** [comment]
I have got that down now since I've found that the plugin couldn't find any head lists (important keyword - most of the time, it should return an empty list instead) for the specified entity at the time so I'm making it print out what it couldn't find.

I'll just be doing the final touch-ups for SNAPSHOT-2 then release it on Github so you're able to see what's wrong.

**Comment by LoneDev6** [comment]
Thanks for the time, let me know :D

Il giorno sab 16 mag 2020 alle ore 09:57 TM (Holly) <
notifications@github.com> ha scritto:

**Comment by Thatsmusic99** [comment]
Try giving it a shot with this snapshot and let me know what it says: https://github.com/Thatsmusic99/HeadsPlus/releases/tag/v6.10.5-SNAPSHOT-2

**Comment by Thatsmusic99** [comment]
In the end, the error was reported a second time and I believe that led me to identify the cause of it; when using /hp reload, do you notice any errors in the console? In which case I'd recommend resetting the customheads.yml file if it is using the default configuration - otherwise, set the config's version to 2.9 with the build above.

**Comment by Thatsmusic99** [comment]
Should be fixed in the latest release of v6.10.5 on Spigot - let me know if the problem arise again and I'll reopen this issue.
